docs/content/integrations/dbt/using-dbt-with-dagster/load-dbt-models.mdx

```diff
@@ -73,32 +69,27 @@ Let's take a look at the arguments we've supplied:
 
 Next, you'll define the code location for your Dagster project. A [code location](/concepts/code-locations), created using the <PyObject object="Definitions" /> object, is a collection of definitions in a Dagster project, such as assets, resources, and so on.
 
-Assets loaded from dbt require a dbt resource, which is responsible for firing off dbt CLI commands. Using the <PyObject module="dagster_dbt" object="DbtCliClientResource" /> resource, we can supply a dbt resource to the dbt project.
+Assets loaded from dbt require a dbt resource, which is responsible for firing off dbt CLI commands. Using the <PyObject module="dagster_dbt" object="DbtCli" /> resource, we can supply a dbt resource to the dbt project.
 
 Open the `__init__.py` file, located in `/tutorial_template/tutorial_dbt_dagster`, and add the following code:
 
 ```python
 import os
 
-from dagster_dbt import DbtCliClientResource
+from dagster_dbt import DbtCli
 from tutorial_dbt_dagster import assets
-from tutorial_dbt_dagster.assets import DBT_PROFILES, DBT_PROJECT_PATH
+from tutorial_dbt_dagster.assets import DBT_PROJECT_PATH
 
 from dagster import Definitions, load_assets_from_modules
 
-resources = {
-    "dbt": DbtCliClientResource(
-        project_dir=DBT_PROJECT_PATH,
-        profiles_dir=DBT_PROFILES,
-    ),
-}
+resources = {"dbt": DbtCli(project_dir=DBT_PROJECT_PATH)}
 
 defs = Definitions(assets=load_assets_from_modules([assets]), resources=resources)
 ```
 
 Let's take a look at what's happening here:
 
-- In the `resources` key, we've provided configuration info for the <PyObject module="dagster_dbt" object="DbtCliClientResource" /> resource.
+- In the `resources` key, we've provided configuration info for the <PyObject module="dagster_dbt" object="DbtCli" /> resource.
```

Comment:
I don't believe `DbtCli` is in our API docs yet, should double check that this link works.

Reply:
Ah good point. I updated this PR to make that change: https://github.com/dagster-io/dagster/pull/15068.